### PR TITLE
BCTimer new addon

### PR DESCRIPTION
**Pull request by KenshiDRK**
This addon creates a timer everytime you enter a BC and stops it when you enter.

### CODE REVIEW

File: addons/BCTimer/BCtimer.lua [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of BCTimer nor the
+      names of its contributors may be used to endorse or promote products
+      derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL KENSHI BE LIABLE FOR ANY
```
Comment (z16) [inline addons/BCTimer/BCtimer.lua:19]: Your name should be spelled in proper capitalization here. I don't know if that's necessary or not, but it's customary. @IryokuChevalier may know more about it.

File: addons/addons.xml [hunk 2]
```diff
@@ -74,6 +74,13 @@
 	<support>http://mibbit.com/#windower@irc.ffochat.com</support>
   </addon>
   <addon>
+    <name>BCtimer</name>
+    <author>Kenshi</author>
+    <description>Creates a timer everytime you enter a BC.</description>
+	<bugtracker>https://github.com/KenshiDRK/Lua/issues</bugtracker>
```
Comment (z16) [inline addons/addons.xml:80]: For some reason you used tabs here.

**Comment by z16** [comment]
You mixed this commit in with your TargetInfo update. Please keep them separate so we can review/pull them independently. You can do this by creating a new branch for each commit, forked off the original Windower repository.

**Comment by KenshiDRK** [comment]
yea I noticed, I'm very new on github so Im still getting to it, I did a new branch but for some reason it mixed them.

**Comment by z16** [comment]
You probably branched off the branch that already had the TargetInfo commit. You need to make a new branch off the original repository.

**Comment by KenshiDRK** [comment]
It seems I commited in my forked 4.1 dev branch, so thats why, should I just remove the entire repository and start again?

**Comment by z16** [comment]
Yeah, you can copy the TargetInfo changes and this change, then close both requests, fork our 4.1-dev branch twice under different branch names and then commit each change to each branch.

**Pull request closed without merging**